<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyQT Sinuses</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonPlot">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>751</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxHarmonics">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>221</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Sinusoidal Harmonics</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutHarmonics">
      <item row="4" column="1">
       <widget class="QLabel" name="labelH6">
        <property name="text">
         <string>6th harmonic, %</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxHarm2">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelH2">
        <property name="text">
         <string>2nd harmonic, %</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxHarm3">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelH3">
        <property name="text">
         <string>3rd harmonic, %</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxHarm4">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelH4">
        <property name="text">
         <string>4th harmonic, %</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxHarm5">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelH5">
        <property name="text">
         <string>5th harmonic, %</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxHarm6">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxHarm7">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelH7">
        <property name="text">
         <string>7th harmonic, %</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxHarm8">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="labelH8">
        <property name="text">
         <string>8th harmonic, %</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxHarm9">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="labelH9">
        <property name="text">
         <string>9th harmonic, %</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxStyle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Line Style</string>
    </property>
    <widget class="QListWidget" name="listWidgetStyles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBoxA">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw black &quot;base&quot; sinus</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxOffset">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Offset</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>241</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelTopText">
        <property name="text">
         <string>TopText</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMidText">
        <property name="text">
         <string>MidText</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QwtKnob" name="Knob0"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBoxAngCos">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelOffsetAs">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset as</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCenter">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPlot">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot: </string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton0">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>390</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>w</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarDummy">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxB">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw yellow &quot;phase-to-phase</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_window"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionNew_window">
   <property name="text">
    <string>New window...</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtKnob</class>
   <extends>QWidget</extends>
   <header>qwt_knob.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
